docs/samples/Microsoft.ML.Samples/Dynamic/FeatureContributionCalculationTransform.cs

```diff
@@ -58,20 +58,22 @@ public static void FeatureContributionCalculationTransform_Regression()
             // Create a Feature Contribution Calculator
             // Calculate the feature contributions for all features
             // And don't normalize the contribution scores
-            var args = new FeatureContributionCalculationTransform.Arguments()
+            var args = new FeatureContributionCalculatingTransformer.Arguments()
             {
                 Top = 11,
                 Normalize = false
             };
-            var featureContributionCalculator = FeatureContributionCalculationTransform.Create(mlContext, args, transformedData, model.Model, model.FeatureColumn);
+
+            var featureContributionCalculator = new FeatureContributionCalculatingTransformer(mlContext, model.Model, model.FeatureColumn, args);
```

Comment:
In this sample, can you give an example where the `FeatureContributionCalculatingTransform` is part of the pipeline and model? This is a common use case -- sometimes even as a mid-learning featurization step.

Reply:
Will do!

---
In reply to: [239965689](https://github.com/dotnet/machinelearning/pull/1847#discussion_r239965689) [](ancestors = 239965689)